sydent/sms/openmarket.py

```diff
@@ -30,6 +30,18 @@
 # Useful for testing.
 #API_BASE_URL = "http://smsc-cie.openmarket.com/sms/v4/mt"
 
+TONS = {
```

Comment:
I've added a comment - it should probably stay in all caps since it's a constant so not sure how to clarify the formatting